src/buildMathML.js

```diff
@@ -259,7 +259,9 @@ groupTypes.styling = function(group, options) {
     };
 
     const newStyle = styleMap[group.value.style];
-    const newOptions = options.havingStyle(newStyle);
+    const newOptions = group.value.mathStart ?
+                       options.havingStyle(newStyle).withMathMode() :
```

Comment:
I would've expected `options.withMathMode()` to be sufficient for this case.

Reply:
Not sure I follow this. `options.withMathMode()` instead of `options.havingStyle(newStyle)`?